### PR TITLE
Add final to all classes.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Blog/Entity/Comment.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
  *     }
  * )
  */
-class Comment
+final class Comment
```
Comment (roxblnfk) [inline src/Blog/Entity/Comment.php:27]: It is bad idea to do finalized entity classes when you are using cycle orm

Comment (terabytesoftw) [reply src/Blog/Entity/Comment.php:27]: `Cycle/Orm` extends the identity classes, it's added in the docs, and I change it.

Comment (terabytesoftw) [reply src/Blog/Entity/Comment.php]: fixed.

**Comment by samdark** [comment]
👍

**Pull request merged**